lib/identity/stateTransitions/identityCreateTransition/validateIdentityCreateSTDataFactory.js

```diff
@@ -32,7 +37,45 @@ function validateIdentityCreateSTDataFactory(dataProvider) {
       result.addError(new IdentityAlreadyExistsError(identityCreateTransition));
     }
 
-    // TODO: Here we need to fetch lock transaction, extract pubkey from it and verify signature
+    if (!result.isValid()) {
+      return result;
+    }
+
+    // fetch lock transaction, extract pubkey from it and verify signature
+
+    const lockedOutBuffer = Buffer.from(identityCreateTransition.getLockedOutPoint(), 'base64');
```

Comment:
This is a part of `validateLockTransaction` let's move it there too. Signature validation as well